.github/workflows/firebase-preview.yml

```
@@ -20,13 +24,67 @@
-          expires: 7d
+          target: barterverse-auto-deploy
+          expires: 7d
+        env:
+          FIREBASE_CLI_EXPERIMENTS: webframeworks
+          GOOGLE_API_KEY: ${{ env.GOOGLE_API_KEY }}
+      
+      - name: Comment on PR with deployment info
+        if: success()
+        uses: actions/github-script@v7
+        with:
+          script: |
```

actionlint (1.7.7)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)